pkg/mongodb/mongodb.go

```diff
@@ -13,31 +14,49 @@ import (
 
 const WaitTimeOutSeconds = 2
```

Comment:
Can we unexport this one as well?

Reply:
Oops, I just realized this constant used nowhere. Maybe we can remove it.